compiler/src/iree/compiler/Codegen/Passes.td

```diff
@@ -257,6 +257,9 @@ def TileAndDistributeToWorkgroups :
     Option<"maxWorkgroupParallelDims", "max-workgroup-parallel-dims", "int32_t",
       /*default=*/ "",
       "Maximum number of dims to distribute workgroups across.">,
+    Option<"skipDistributionLoops", "skip-distribution-loops", "bool",
```

Comment:
Instead of `bool` option, I dont know if it is possible, but can we accept the `linalg::DistributionMethod` enum directly? I guess we will have to pass the magic number and the interpret it as an enum (but maybe including the definition allows you to just use the enum type directly).

Reply:
I tried to make the option `linalg::DistributionMethod` but then I don't see a way to pass the value as command line. I don't see a simpler solution. Let me know if you see something convenient.

Reply:
yeah, from command line you have to pass the enum integer value. Thats just for testing... through the `create` API you can pass the enum directly?

Reply:
changed the comment line to take an int then and cast the enum in the pass. Is that what you had in mind?